tests/FSharp.Compiler.ComponentTests/Conformance/BasicGrammarElements/CustomAttributes/CallerArgumentExpression/CallerArgumentExpression.fs

```diff
@@ -0,0 +1,384 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace Conformance.BasicGrammarElements
+
+open Xunit
+open FSharp.Test.Compiler
+open FSharp.Test
+
+module CustomAttributes_CallerArgumentExpression =
+
+    [<FactForNETCOREAPP>]
```

Comment:
Is there a particular reason for all added tests being `FactForNETCOREAPP` and not `Fact` that would also test Desktop framework?